src/docs/user/references/DataConnectors.adoc

```diff
@@ -100,13 +103,20 @@ The URL connector allows, in addition, to import data using HTTP and HTTPS proto
 | false
 |===
 
-*How to use this task:* We highly recommend the user to not provide his password within the URL and to instead use the third-party credential mechanism as follows: `{key: <PROTOCOL>://<username>@<hostname>, value: <PROTOCOL>_PASSWORD}`. Please refer to the User documentation to learn how to add link:../user/ProActiveUserGuide.html#_third_party_credentials[third-party credentials].
+*How to use this task:*
+
+We highly recommend the user to not provide his password within the URL and to instead use the third-party credential mechanism as follows: `{key: <PROTOCOL>://<username>@<hostname>, value: <PROTOCOL>_PASSWORD}`. Please refer to the User documentation to learn how to add link:../user/ProActiveUserGuide.html#_third_party_credentials[third-party credentials].
 
 === SQL
-The SQL connectors allow to import data from Relational DataBase Management Systems (RDBMS).
+The SQL connectors allow to import and export data from and to Relational DataBase Management Systems (RDBMS).
 Currently, we have connectors for Mysql, Oracle, Postgres, Greenplum and Sql server.
```

Comment:
Not part of your changes but, do we purposefully use these simplified names for the different RDBMS instead of their official names? eg. MySQL, PostgreSQL or SQL Server?

Reply:
You are right ! this dates back to when denis wanted everything in lowercase and short...
We started using full official names but we didn't update this part.
I'll do at once